docs/MAINTAINING.md

```diff
@@ -0,0 +1,40 @@
+# Maintaining LoopBack
+
+Congratulations! Since you're reading this page, you are probably already a
+maintainer of this repo or close to be one. Thank you for your contributions so
+far!
+
+## Why do I want to be a maintainer?
+
+- Greater influence on LoopBack's future direction
+- Commit (write) rights to `loopback-next` repo
+- Ability to review and land pull requests, edit/categorize/close issues
+- Access to a private Gitter room for LoopBack maintainers
+
+## What are a maintainer's responsibilities?
+
+We ask you to follow the existing processes, see
+http://loopback.io/doc/en/contrib/Governance.html. This means mostly:
+
+- Be nice to others
+- Try to reach consensus with other maintainers before making a decision
+- Always use pull requests to make code changes
+- Honour our current conventions, see
+  http://loopback.io/doc/en/contrib/triaging-pull-requests.html and
+  http://loopback.io/doc/en/contrib/style-guide.html (but feel free to propose
+  changes to these conventions!)
+- To avoid possible confusion, we don't have any specific requirements about the
+  amount of time you should spend on the project - just keep working on things
+  that you find important to you, in a pace that suits you. We may ask you to
+  take a look at issues and pull requests related to code you contributed
+  yourself.
+
+## More questions?
```

Comment:
If someone has a question that isn't covered here, should we also include something like "If you have more questions, feel free to reach out by * _insert way for them to reach us_ *"? Also, how do non-IBM maintainers communicate with the team outside GitHub issues/PRs?

Reply:
That's a good question! I guess the simplest way is to create a github issue.  Another way is to email the maintainers (email might be found in some of our github profiles), but I'd rather not encouraging people to email us. :) 

@strongloop/loopback-maintainers, any suggestions?

Reply:
@nabdelgadir, I've added the last "question" regarding your comment. Thanks!

Reply:
The questions section LGTM 👍

Reply:
FWIW, GitHub offers teams a place for private discussions, see e.g. here: https://github.com/orgs/strongloop/teams/loopback-maintainers/discussions

I am not sure how do notifications work for discussions, using GitHub issues may be a safer way.